plugins/woocommerce/src/Admin/Features/ProductBlockEditor/ProductTemplates/ProductTemplateInterface.php

```diff
@@ -10,6 +10,31 @@
  */
 interface ProductTemplateInterface {
 
+	/**
+     * General group ID.
+     */
+    const GENERAL_GROUP = 'group/general';
```

Comment:
This does not feel like the best place for these, but unfortunately PHP versions <8.2 don't support constants in traits and I don't think these core-specific groups/sections/fields belong in the abstract class.